### PR TITLE
Create directory structure for super app

### PR DESCRIPTION
**Pull request by glencbz**
Closes https://github.com/bettersg/call-home/issues/74
(This is also the final commit's message)

The eventual goal is to have something like:

```
.
├── apps/
│   ├── common
│   └── super-app
│       ├── pro-bono
│       ├── personal-calls
│       └── twc2
├── services/
│   ├── superset
│   ├── strapi
│   └── personal-calls
└── infra
```

This should make it relatively simple to:

- deploy a single frontend app
- develop sub-apps in the super app while sharing code
- develop services in isolation
- make changes to cloud infrastructure

We want to split the personal call code into app + service, but that's going to
be a big effort. Keep it in the top-level personal-calls/ in the meantime.

If we need to share frontend code with the personal calls app, some options are:

- Duplicate the code
- Create a local npm module for the shared code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1 @@
-node_modules/
-.log/
+.npm/
```
Comment (chaychoong) [inline .gitignore:1]: hmmm newline things

**Review by halfwhole — COMMENTED** [review]
the directory structure looks great! everything pretty much makes sense to me except for the Makefile (sorry) so can't comment about that, think I might go read up on that sometime. would also be nice to hear a bit more about the pre-existing structure another time

File: personal-calls/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+- Call a user-specified number
+- Save user preferences to the DB
+
+## How to run
+
+### Setup
+
+- [TypeScript](https://www.npmjs.com/package/typescript)
+- node 12 (preferably installed with versioning tool such as [Node Version Manager](https://github.com/nvm-sh/nvm))
```
[the remark this answers is not in the document]
Comment (glencbz) [reply personal-calls/README.md:13]: Yeah that's right. Good catch.

File: personal-calls/package.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "call-home-mvp",
+  "version": "1.0.0",
+  "description": "This is very much in the MVP stage.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/glencbz/ring-a-senior.git"
```
[the remark this answers is not in the document]
Comment (glencbz) [reply personal-calls/package.json:7]: (whoops)

File: .gitignore (hunk 1 above)
Comment (glencbz) [reply .gitignore:1]: Bleh, thanks.

**Comment by glencbz** [comment]
For our purposes, think of Makefiles as a collection of scripts (like npm scripts). This isn't actually what they were designed for (they're designed for compiling C), but they do a good-enough job of being a collection of scripts since they're pretty portable and it's easy to opt-out of the 'compiling' assumptions (e.g. targets refer to filenames by default).

**Review by halfwhole — APPROVED** [review]

**Pull request merged**